README.md

```diff
@@ -1,23 +1,28 @@
-# con-pca-tasks #
+# Con-PCA Tasks #
 
 [![GitHub Build Status](https://github.com/cisagov/con-pca-tasks/workflows/build/badge.svg)](https://github.com/cisagov/con-pca-tasks/actions)
 
-This is a generic skeleton project that can be used to quickly get a
-new [cisagov](https://github.com/cisagov) GitHub project started.
-This skeleton project contains [licensing information](LICENSE), as
-well as [pre-commit hooks](https://pre-commit.com) and
-[GitHub Actions](https://github.com/features/actions) configurations
-appropriate for the major languages that we use.
+## Description ##
 
-In many cases you will instead want to use one of the more specific
-skeleton projects derived from this one.
+This repo contains the tasks portion for Con-PCA, Continuous Phishing
+Assessment. This application is reponsible for executing scheduled tasks for
+Con-PCA at scale.
 
-## New Repositories from a Skeleton ##
+## Related Con-PCA Repositories ##
 
-Please see our [Project Setup guide](https://github.com/cisagov/development-guide/tree/develop/project_setup)
-for step-by-step instructions on how to start a new repository from
-a skeleton. This will save you time and effort when configuring a
-new repository!
+- [con-pca-api](https://github.com/cisagov/con-pca-api)
+- [con-pca-cicd](https://github.com/cisagov/con-pca-cicd)
+- [con-pca-web](https://github.com/cisagov/con-pca-web)
+
+## Commands ##
+
+Run application
+
+- `make run`
+
+Test application
+
+- `make test`
```

Comment:
I like this flow a bit more as it sets up for more involved command usage possibilities. I'm not 100% sure this will correctly merge through the GitHub web UI as fair warning.

```suggestion
### Running the application ###

```console
make run
```

### Testing the application ###

```console
make test
```
```

Reply:
@mcdonnnj resolved with [6ab301d](https://github.com/cisagov/con-pca-tasks/pull/1/commits/6ab301d53b29c3ef056462a63f4656455bb9a9fa)